CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
 ## XX.XX.XX - 20XX-XX-XX
+### PaymentSheet
+- [ADDED] Support for card brand filtering in private beta.
```

Comment:
We typically link a PR here, but I think it doesn't add much value, so I'm fine omitting it.